portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> messages;
```

Comment:
Use interfaces whenever possible. Also, this should probably be private (not specifying public/private, by default, makes it "package-private" meaning it's visible to all classes within com.google.sps.servlets).

`private List<String> messages;`

Reply:
If I've addressed these comments in a different Pull Request, should I just merge the other branch into this one so that the changes I made are here, or should I wait for the other PR to be submitted and merge from master, as that would not have commits that have not been checked by you guys. Unsure of the best course of action in cases like this.